openmetadata-ui/src/main/resources/ui/src/utils/Query/QueryUtils.ts

```diff
@@ -51,3 +52,16 @@ export const parseSearchParams = (param: string) => {
 export const stringifySearchParams = (param: QuerySearchParams) => {
   return Qs.stringify(param);
 };
+
+export const INVALID_SQL_ERROR = 'SQL Invalid';
```

Comment:
Can we use localisation here?

Reply:
will not need this text more, after replacing the html form with antDForm